packages/core/src/log4js.classes.ts

```diff
@@ -54,4 +58,11 @@ export class Log4jsLogger implements LoggerService {
   getTimestamp() {
     return Log4jsLogger.getTimestamp();
   }
+
+  printStackTrace(trace?: string) {
+    if(!trace) {
+      return;
+    }
+    this.logger.trace(trace);
```

Comment:
@sosohime  Consider using `logger.error(trace)` instead of `logger.trace(trace)` and renaming arguments `trace` to `stackTrace`.

since log4js official log level is `(ALL < TRACE < DEBUG < INFO < WARN < ERROR < FATAL < MARK < OFF `. We suppose not using TRACE-LEVEL (with log-level setting = `INFO` in production, `trace` level logs will not print)

reference: https://log4js-node.github.io/log4js-node/api.html

Reply:
@aquariuslt ur right，to maintain the same behavior as nestjs logging, needs to ensure errror trace output is correct ,we should use `ERROR` level to print  error stack trace
reference: https://github.com/nestjs/nest/blob/a2f7761275e4613722aeaf654e487728729431b3/packages/common/services/logger.service.ts#L179